Resources/Prototypes/_StarLight/Roles/Jobs/Medical/surgeon.yml

```
@@ -21,4 +21,9 @@
     ears: ClothingHeadsetMedical
     belt: ClothingBeltMedicalFilled
   inhand:
-    - ClothingBackpackDuffelSurgeryFilled
\ No newline at end of file
+    - ClothingBackpackDuffelSurgeryFilled
+
+- type: chameleonOutfit
+  id: SurgeonChameleonOutfit
+  job: Surgeon
+  equipment: {} # may someone tell me what these are supposed to look like when chameleon'd
```

```suggestion
  equipment:
    head: ClothingHeadNurseHat
    eyes: ClothingEyesHudMedical
    mask: ClothingMaskSterile
    neck: ClothingNeckStethoscope
    outerClothing: ClothingOuterCoatLab
    gloves: ClothingHandsGlovesLatex
```
exactly the same as a doctor